install/db/dbcreate.in

```diff
@@ -15,6 +15,17 @@ echo "*** Setting up the FOSSology database ***"
 # At some point this is where we could dynamically set the db password
 
 # first check that postgres is running
+psqln=0
+until [ "$psqln" -ge 10 ]; do
+      su postgres -c 'echo \\q|psql'
+      if [ $? = 0 ]; then
+	  break
```

Comment:
Woops, an indentation issue. Please change the tabs to spaces to keep the file consistent.

Reply:
fixed. Thanks.